amplify/backend/function/risestandstrongcb34053a/src/routes/shifts.js

```diff
@@ -194,11 +195,22 @@ router.get('/:startTimestamp', async (req, res) => {
  *      responses:
  *         "200":
  *            description: Success
- *         "404":
+ *         "400":
  *            description: startTimestamp not found
  */
 router.put('/:startTimestamp', async (req, res) => {
-   res.end();
+   const startTimestamp = req.params.startTimestamp;
+   const shiftBody = {
+      startTimestamp: parseInt(startTimestamp),
+      ...req.body
+   }
```

Comment:
I think we should add a check to this code so that if the user passes us something like "hello" as startTimestamp we don't try and add that invalid object to the shifts table. You can do so by checking it parseInt returns NaN